### DISCUSSION TITLE
Can a value be set on a Variable Font's set of available AXIS(s), programmatically, using managed code (C#)?

### DISCUSSION THREAD

**Discussion by GUI-Kenny** [microsoft/WindowsAppSDK#1559] [Feedback wanted]
Can a value be set on a Variable Font's set of available AXIS(s), programmatically, using managed code (C#)? Like in a TextBlock control used in XAML.

I know how to do this with Fonts in CSS and HTML5, but don't know if .NET5 (WinUI 3) is set up to handle AXIS(s) from Variable Fonts. I also know, there is an interface, DWriteCore, where I can get AXIS information, and even functions to draw to a Device Context using these values.

Just want to set up a Slider to adjust Axis(s) and display the results.

I'm trying to see, where this capability is in Managed Code (C#).

Thank you in advance, for any assistance.

**Comment by nickrandolph** [comment]
@GUI-Kenny I'm guessing this is probably more of a question for [Windows UI](https://github.com/microsoft/microsoft-ui-xaml) - you might want to cross-post this question there.

**Reply by GUI-Kenny** [reply]
I don't see a 'Discussion" tab on https://github.com/microsoft/microsoft-ui-xaml to post a question.
